src/unix/core.c

```diff
@@ -63,6 +63,7 @@
 # include <sys/filio.h>
 # include <sys/ioctl.h>
 # include <sys/wait.h>
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
I'll move it over. I had it there originally but noticed that limits.h was not being included in `src/unix/fs.c` but in core.c, so I thought if it was included there 

Other bits like IOV_MAX are only in syslimits on freeBSD.

I'll drop the first commit and move this all into fs.c